generic/src/test/java/org/datahem/processor/generic/GenericStreamPipelineTest.java

```diff
@@ -0,0 +1,600 @@
+        fields.add(new TableFieldSchema().setName("repeatedInt32").setType("INTEGER").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedInt64").setType("INTEGER").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedDouble").setType("FLOAT").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedFloat").setType("FLOAT").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedBool").setType("BOOLEAN").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedBytes").setType("BYTES").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("repeatedEnum").setType("INTEGER").setMode("REPEATED").setDescription("").setCategories(fieldCategories));
+        fields.add(new TableFieldSchema().setName("_ATTRIBUTES").setType("RECORD").setMode("REPEATED").setDescription("").setFields(assertAttributesSchema.getFields()));
+        assertSchema.setFields(fields);
+
+        TableSchema eventSchema = null;
+        String tableDescription = "";
+        try{
+            ProtoDescriptor protoDescriptor = ProtobufUtils.getProtoDescriptorFromCloudStorage("mathem-ml-datahem-test-descriptor", "testSchemas.desc");
+            Descriptor descriptor = protoDescriptor.getDescriptorByName("datahem.test.TestWithoutOptions");
+            eventSchema = ProtobufUtils.makeTableSchema(protoDescriptor, descriptor, ".*590903188537942776.*");
+            Assert.assertEquals(eventSchema, assertSchema);
+            LOG.info("withoutOptionsTest assert TableSchema without errors.");
+        }catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        try{
+
+            PCollection<TableRow> output = p
+                .apply(Create.of(Arrays.asList(pm)))
+                .apply(ParDo.of(new GenericStreamPipeline.PubsubMessageToTableRowFn(
+                    StaticValueProvider.of("mathem-ml-datahem-test-descriptor"),
+                    StaticValueProvider.of("testSchemas.desc"),
+                    StaticValueProvider.of("datahem.test.TestWithoutOptions"))));
+            PAssert.that(output).containsInAnyOrder(assertTableRow);
+            p.run();
+            LOG.info("withoutOptionsTest assert TableRow without errors.");
+        }catch (Exception e) {
+            LOG.error(e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+
+
+    @Test
+	public void withSchemaOptionsTest(){
+
+        //BEGIN INPUT
+        JSONObject messageChild = new JSONObject()
+            .put("StringField", "a string")
+            .put("Int32Field", 32) 
+            .put("Int64Field", 64) 
```

Comment:
Why not a real int64 like 79239016081? So that we really make sure it's not limited to 32 bits...